vendor.conf

```diff
@@ -28,7 +28,7 @@ github.com/imdario/mergo                            v0.3.7
 github.com/konsorten/go-windows-terminal-sequences  v1.0.3
 github.com/matttproud/golang_protobuf_extensions    v1.0.1
 github.com/Microsoft/go-winio                       v0.4.14
-github.com/Microsoft/hcsshim                        v0.8.10
+github.com/Microsoft/hcsshim                        77c0270424049ab4c850c076601add11882e7c1e
```

Comment:
Do we not have a tag that contains the work?

Reply:
0.9.x should have the appending of layer hashes, however, that implementation reads entire layer in memory and doesn't support passing an io.Reader. If we cut a 0.10 release, I'd be happy to use that, but I don't think it's happening yet?

Reply:
Ahhh, gotcha. Going by commit is likely fine for now then. Whenever we cut a tag we can update